### DISCUSSION TITLE
Total number of unique avatars

### DISCUSSION THREAD

**Discussion by dzoba** [dicebear/dicebear#189] [Q&A] [answered]
It would be helpful to know, for each category, the total number of unique avatars available for that category.

**Comment by FlorianKoerner** [comment, marked as answer]
# Version 5.x
https://dicebear.com/guides/how-many-unique-avatars

# Version 5.x Alpha
I have calculated how many unique avatars can be created with version 5.0.0-alpha.15.

| Package                      | Number of unique avatars |
|------------------------------|--------------------------|
| @dicebear/adventurer         | 4.419.792.000            |
| @dicebear/adventurer-neutral | 280.800                  |
| @dicebear/avataaars          | 7.191.699.540.480        |
| @dicebear/avataaars-neutral  | 13.104                   |
| @dicebear/big-ears           | 1.207.664.640.000        |
| @dicebear/big-ears-neutral   | 3.594.240                |
| @dicebear/big-smile          | 4.792.320                |
| @dicebear/bottts             | 9.049.320                |
| @dicebear/bottts-neutral     | 2.394                    |
| @dicebear/croodles           | 105.753.600              |
| @dicebear/croodles-neutral   | 2.592                    |
| @dicebear/fun-emoji          | 1.350                    |
| @dicebear/icons              | 2.774                    |
| @dicebear/identicon          | 319.333                  |
| @dicebear/micah              | 1.119.744.000.000        |
| @dicebear/miniavs            | 145.800                  |
| @dicebear/open-peeps         | 76.576.500               |
| @dicebear/personas           | 27.546.624               |
| @dicebear/pixel-art          | 15.581.479.914.000.000   |
| @dicebear/pixel-art-neutral  | 5.819.413.600            |

You can find the calculation here:
https://docs.google.com/spreadsheets/d/1pm8X3XUR3UOgXz7THbnwy94mWH-JQUWC8ZPHMHyKqAc/edit?usp=sharing

Calculations without warranty of their correctness.
